pkg/collector/corechecks/system/wincrashdetect/probe/crashdump.cc

```diff
@@ -0,0 +1,67 @@
+#include "crashdump.h"
+#include "_cgo_export.h"
```

Comment:
What is _cgo_export.h for?

Reply:
in includes the forward declarations that cgo generates for functions implemented in `go`, specifically the `logLineCallback`